### PR TITLE
New recipe cadmesh

### PR DESCRIPTION
**Pull request by skoch9**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/cadmesh/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "cadmesh" %}
+{% set version = "v0.1" %}
```
Comment (chrisburr) [inline recipes/cadmesh/meta.yaml]: Conda versions shouldn't start with `v`:
```suggestion
{% set version = "0.1" %}
```
(I've suggested a change on the URL to match)

File: recipes/cadmesh/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/skoch9/cadmesh/archive/{{ version }}.tar.gz
```
Comment (chrisburr) [inline recipes/cadmesh/meta.yaml]:
```suggestion
  url: https://github.com/skoch9/cadmesh/archive/v{{ version }}.tar.gz
```

File: recipes/cadmesh/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: 
+  - {{ PYTHON }} -m pip install . -vv
+
+
+requirements:
+  build:
```
Comment (chrisburr) [inline recipes/cadmesh/meta.yaml]: If there are no build requirements this line should be removed:
```suggestion
```

File: recipes/cadmesh/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  host:
+    - python >=3.0
+    - pip
+  run:
+    - python >=3.0
+    - numpy
+
+test:
+  imports:
+    - numpy
```
Comment (chrisburr) [inline recipes/cadmesh/meta.yaml]: Should this be:
```suggestion
    - cadmesh
```

**Comment by marcelotrevisani** [comment]
ping @skoch9

**Pull request closed without merging**